x-pack/plugins/security_solution/public/common/components/navigation/breadcrumbs/index.ts

```diff
@@ -186,18 +186,7 @@ export const getBreadcrumbsForRoute = (
     if (spyState.tabName != null) {
       urlStateKeys = [...urlStateKeys, getOr(tempNav, spyState.tabName, object.navTabs)];
```

Comment:
urlStateKeys is no longer used... was this intended @semd?

Reply:
nope, cleaning it 👍